hack/validation/labels.sh

```diff
@@ -4,7 +4,7 @@
 ## checking for restricted labels while filtering out well-known labels
```

Comment:
This looks great! Thanks for the fix here! Do you mind adding tests to `nodeclaim_validation_cel_test.go` and `nodeclaim_validation_webhook_test.go` so that we make sure we are covering cases where someone passes through a sub-domain of any of the labels that are exceptions?

Reply:
tests added https://github.com/aws/karpenter-core/pull/808/commits/515bcee4f95b9948678974245f5c8a64a9aed69b
+ updated CRDs CEL validation to allow those subdomains https://github.com/aws/karpenter-core/pull/808/commits/7a5bf33b105f1663982c49a047f24418733ac1d2